distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -22,6 +22,11 @@ a {
 .global-wrapper {
   margin: 0 auto;
   max-width: 1400px;
+  padding: 10px;
+
+  .filtered & {
+    padding: 10px 0 0 $filters-width;
```

Comment:
Does it need the `$filters-width` left padding? It'll appear to the right of them either way because `.filtered-charts` is `display: flex;`.